Vostok.ClusterClient.Singular/Helpers/RequestParametersExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using JetBrains.Annotations;
+using Vostok.Clusterclient.Core.Model;
+
+namespace Vostok.Clusterclient.Singular.Helpers
+{
+    [PublicAPI]
+    public static class RequestParametersExtensions
+    {
+        internal const string RequestParametersTagsFilterKey = "ReplicasTagsFilter";
+
+        /// <summary>
+        /// <para> Sets given <paramref name="filterString" /> replicas filtering expression in string format to <paramref name="requestParameters" />. </para>
+        /// <para> This expression will be sent via request headers and executed in Singular. </para> 
+        /// <para> If <paramref name="filterString" /> returns false then replica will be filtered.</para>

```

Comment:
It is worth clarifying that "an expression derived from a filterString."

Reply:
done